src/format-files.js

```diff
@@ -49,9 +51,10 @@ function formatFilesFromArgv({
     prettierLast,
     prettierOptions,
   }
-
+  const configContext = new Config({}, new Linter())
   if (eslintConfigPath) {
-    prettierESLintOptions.eslintConfig = ConfigFile.load(eslintConfigPath)
+    prettierESLintOptions.eslintConfig =
+      ConfigFile.load(eslintConfigPath, configContext)
```

Comment:
There's nothing that's changing about the external API right? So this is technically not a breaking change I think.